hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2703,17 +2703,15 @@ func reconcileCAPIManagerRole(role *rbacv1.Role) error {
 				"hostedcontrolplanes",
 				"hostedcontrolplanes/status",
 			},
-			Verbs: []string{"*"},
+			Verbs: []string{"get", "list", "watch", "delete"},
```

Comment:
Looks like you at least need patch and update

Reply:
Just updated it--thanks for looking into that!